extensions/Worker.Extensions.Http.AspNetCore/src/FunctionContextExtensions.cs

```diff
@@ -17,17 +17,15 @@ public static class FunctionContextExtensions
         /// </summary>
         /// <param name="context">The <see cref="FunctionContext"/>.</param>
         /// <returns></returns>
-        public static HttpContext GetHttpContext(this FunctionContext context)
+        public static HttpContext? GetHttpContext(this FunctionContext context)
```

Comment:
nit: Update the returns part of method summary since this is going to be a pubic extns method.